packages/datetime/src/shortcuts.tsx

```diff
@@ -119,11 +135,11 @@ function createDefaultShortcuts(allowSingleDayRange: boolean, hasTimePrecision:
 
     return [
         ...singleDayShortcuts,
-        createShortcut("Past week", [oneWeekAgo, today]),
-        createShortcut("Past month", [oneMonthAgo, today]),
-        createShortcut("Past 3 months", [threeMonthsAgo, today]),
-        createShortcut("Past 6 months", [sixMonthsAgo, today]),
-        createShortcut("Past year", [oneYearAgo, today]),
-        createShortcut("Past 2 years", [twoYearsAgo, today]),
+        createShortcut(useSingleDateLabels ? "1 week ago": "Past week", [oneWeekAgo, today]),
```

Comment:
how does i18n work with these?

Reply:
... it doesn't. would file a separate issue for that